packages/mobile-first-storybook/src/components.stories.tsx

```diff
@@ -1,44 +1,164 @@
-    iconUnselectedColor={'#ccc'}
-    iconHeight={18}
-    iconWidth={20}
-    iconSelectedColor={'#FF00FF'}
-    title={'Browse'}
-    iconUrl={
-      'https://s3.amazonaws.com/assets.aloompa.com/TestFolder/reservations-tab-icon-browse.svg'
-    }
-  />
-));
-
-storiesOf('TopNav', module).add('with navigation', () => (
-  <TopNav history={[]} routeTitle={'Navigation Title'} navigatBack={() => {}} />
-));
-
-storiesOf('TopNav', module).add('with modal', () => (
-  <TopNav
-    history={[]}
-    routeTitle={'Navigation Title'}
-    navigatBack={() => {}}
-    mode={'modal'}
-  />
-));
+import { linkTo } from '@storybook/addon-links';
+import { TabRouter, TabButton, TopNav } from '@aloompa/mobile-first-components';
+
+storiesOf('TabButton', module)
+  .add('unselected', () => (
+    <TabButton
+      selected={false}
+      iconUnselectedColor={'#ccc'}
+      iconHeight={18}
+      iconWidth={20}
+      iconSelectedColor={'#FF00FF'}
+      title={'Browse'}
+      onPress={linkTo('TabButton', 'selected')}
+      iconUrl={
+        'https://s3.amazonaws.com/assets.aloompa.com/TestFolder/reservations-tab-icon-browse.svg'
+      }
+    />
+  ))
+  .add('selected', () => (
+    <TabButton
+      selected={true}
+      iconUnselectedColor={'#ccc'}
+      iconHeight={18}
+      iconWidth={20}
+      iconSelectedColor={'#FF00FF'}
+      title={'Browse'}
+      iconUrl={
+        'https://s3.amazonaws.com/assets.aloompa.com/TestFolder/reservations-tab-icon-browse.svg'
+      }
+    />
+  ));
+
+storiesOf('TopNav', module)
+  .add('at initial route', () => (
+    <TopNav history={['home']} routeTitle={'Home'} navigateBack={() => {}} />
+  ))
+  .add('with history', () => (
+    <TopNav
+      history={['home', 'other']}
+      routeTitle={'Other Page'}
+      navigateBack={linkTo('TopNav', 'at initial route')}
+    />
+  ))
+  .add('with modal', () => (
+    <TopNav
+      history={[]}
+      routeTitle={'Modal Title'}
+      navigateBack={linkTo('TopNav', 'with history')}
+      mode={'modal'}
+    />
+  ));
+
+const tabViews = [
+  <div style={{ display: 'flex', justifyContent: 'center', width: '100vw' }}>
+    <span>Browsing..</span>
+  </div>,
+  <div style={{ display: 'flex', justifyContent: 'center', width: '100vw' }}>
+    <span>Tickets?</span>
+  </div>,
+  <div style={{ display: 'flex', justifyContent: 'center', width: '100vw' }}>
+    <span>Profile!</span>
+  </div>
+];
+
+const tabConfigA = (nTabs) => ({
+  iconUnselectedColor: '#ccc',
+  iconHeight: 18,
+  iconWidth: 20,
+  iconSelectedColor: '#000',
+  iconUrl:
+    'https://s3.amazonaws.com/assets.aloompa.com/TestFolder/reservations-tab-icon-browse.svg',
+  onPress: linkTo(`TabBar/${nTabs} Tabs`, 'first selected'),
+  title: 'Browse'
+});
+
+const tabConfigB = (nTabs) => ({
+  iconUnselectedColor: '#ccc',
+  iconHeight: 18,
+  iconWidth: 22,
+  iconSelectedColor: '#000',
+  iconUrl:
+    'https://s3.amazonaws.com/assets.aloompa.com/TestFolder/reservations-tab-icon-tickets.svg',
```

Comment:
Should we host these in a more generic spot to not let everyone know our assets bucket? cc: @tysoncadenhead @jottenlips

Reply:
yes, definitely!

Reply:
could we just put them in the components repo?

Reply:
At the very least, we should point to Imgix instead of the bucket like this: https://aloompa-assets.imgix.net/TestFolder/reservations-tab-icon-browse.svg